frontend/components/CollateralTable.vue

```diff
@@ -30,7 +30,8 @@
                     :content="marketUnitPrice"
                     trigger="hover"
                 >
-                    <span class="text-red-500">error</span>
+                    <span class="mr-1 text-red-500">error</span>
+                    <span class="Error">...</span>
```

Comment:
My original suggestion was to show `marketUnitPrice` error text instead of `error` text here, but trimmed, so the user will see beginning of the actual error text (in red) and will realise without any additional signifiers that hovering it will show full text

Reply:
![Screenshot 2022-01-26 at 16 27 01](https://user-images.githubusercontent.com/30908158/151192893-4645d1df-b946-4336-a291-2eaed0a6974a.png)